Source: Ingmar Vanhassel
Upstream: No, will submit
Reason: See below

From bf02f21e610213f9368e0e5947a2cfe6854a19b4 Mon Sep 17 00:00:00 2001
From: Ingmar Vanhassel <ingmar@exherbo.org>
Date: Thu, 18 Jun 2009 20:20:25 +0200
Subject: [PATCH] Use LIB_SUFFIX, allows installing 64bit libs into /usr/lib64.

---
 tools/build/CMake/BoostCore.cmake |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/tools/build/CMake/BoostCore.cmake b/tools/build/CMake/BoostCore.cmake
index cfcb1f4..8517029 100644
--- a/tools/build/CMake/BoostCore.cmake
+++ b/tools/build/CMake/BoostCore.cmake
@@ -730,7 +730,7 @@ macro(boost_library_variant LIBNAME)
       
       # Installation of this library variant
       string(TOLOWER ${PROJECT_NAME} libname)
-      install(TARGETS ${VARIANT_LIBNAME} DESTINATION lib COMPONENT ${LIB_COMPONENT})
+      install(TARGETS ${VARIANT_LIBNAME} DESTINATION lib${LIB_SUFFIX} COMPONENT ${LIB_COMPONENT})
       set_property( 
             TARGET ${VARIANT_LIBNAME}
             PROPERTY BOOST_CPACK_COMPONENT
-- 
1.6.3.2

